pan-domain-auth-play_2-4-0/src/main/scala/com/gu/pandomainauth/action/Actions.scala

```diff
@@ -154,11 +149,11 @@ trait AuthActions extends PanDomainAuth {
 
 
   def readAuthenticatedUser(request: RequestHeader): Option[AuthenticatedUser] = readCookie(request) map { cookie =>
-    CookieUtils.parseCookieData(cookie.value, settings.secret)
```

Comment:
This would never have worked as it is trying to read the new cookie using the legacy secret.